guides/urls.py

```diff
@@ -13,4 +13,5 @@
     path("<int:pk>/delete", views.DeleteView.as_view(), name="delete"),
     path("feed/atom", LatestGuidesAtomFeed(), name="feed_atom"),
     path("feed/rss", LatestGuidesRSSFeed(), name="feed_rss"),
+    path("search/", views.SearchView.as_view(), name='search')
```

Comment:
Do we need another path for this? I think we could just integrate it to the main `IndexView`, seeing as you default to showing all guides if no term was specified.